data/korath/korath hails.txt

```diff
@@ -0,0 +1,1470 @@
+		"exile humanika: You will give us"
+
+phrase "exile mixed: food"
+	phrase
+		"exile: good food" 2
+		"exile humanika: non-luxury foodstuffs"
+
+# Wish list food, probably half joking
+phrase "exile: good food"
+	word
+		"asame' oavip aok lane'"       # fresh dark green lettuce
+		"nrahfot tasah pagrel"         # crunchy rhino beetles
+		"ilehulk' lane' nrag nrakaf"   # ripe succulent strawberries
+		"sud nrannrab ranres nracah pikob" # baked cod with garlic
+		"kemmij emu' nrannrab"         # baked sole fillet
+		"'ah koam imipd"               # Chilean Sea Bass
+		"rati is 'aka 'aklet"          # fish en papillote
+		"rumef rohfe osaf"             # honeydew melons
+		"meba ak'eb nrag pagrel"       # crisp gala apples
+		"oham nrirot nradrid 'omap"    # finely chopped yellow squash
+		"ica' oavip kolab"             # shredded collard greens
+		"kamo bosip nrag kaimennref"   # live squirming mealworms
+		"totnrafe' aqlam nridad nramen" # a bowlful of hornworm larvae
+		"polome' irumut kofe'"         # an entire ant colony
+		"ratobfok la'eh nridad rimim"  # a big heaping pile of waxworms
+		"nrala' lomek nroloh akro"     # a nest of ostrich eggs
+		"igah 'otik rinris ranres ra'aip" # chilled baby mice with garnish
+		"ak'ab lamo midet ranres u'tah nradrid" # a pasta of small snakes with mice meatballs
+
+phrase "exile mixed: raw materials"
+	phrase
+		"exile: good raw materials" 2
+		"exile humanika: raw materials"
+
+phrase "exile: good raw materials"
+	word
+		"ranrakah 'afe irlof"            # pure gold ingots
+		"foirukomb nrag kanra' 'omap"    # highly refined plutonium
+		"ralahfem foirakik"              # titanium sheets
+		"kabfelebe' ruk nrakah rek'nrok" # quarter ton tungsten rods (Korath tons, obviously)
+		"raimleh arlobfe'"               # perfect diamonds
+		"afim pomob kemab anahfek"       # fifty palettes of copper
+		"rabab ogat 'alet"               # hardwood planks
+		"'okale' melah li'ab rakaoh"     # a hundred barrels of manufactured sand
+		"avah maloktolk' ma'af"          # bulk structural steel
+		"ehokurar ruhlat ifama"          # naturally [formed] carbon nanotubes
+		"ralahfem karuhlatimub"          # polycarbonate sheets
+
+phrase "exile mixed: and we will let you live."
+	phrase
+		"exile: and we will let you live." 2
+		"exile humanika: and we will let you live."
+
+phrase "exile humanika: and we will let you live."
+	word
+		"lat 'oo liff."                   # let you live (mispronounced)
+		"oo go frie."                     # you go free (mispronounced)
+		"n't oo die."                     # not you die (mispronounced)
+		"oo stai ali'f."                  # you stay alive (mispronounced)
+		"wee go o o way."                 # we go [our own way|away] (mispronounced)
+		"wee go way."                     # we go away (mispronounced)
+		"uu keep ur skin"                 # you will keep your skin (mispronounced)
+		"u see morrow"                    # you [will] see tomorrow (mispronounced)
```

Comment:
Avoid temporal references.

```suggestion
```

Reply:
Is the "avoid temporal references" a cultural/society thing; or could this be adjusted to something less specific?

As far as that goes, I agree it should be less specific. "u see later" or "u see in time"

That being said, could the Korath simply be miscinstruing the word tpmorrow as meaning "future"?

Reply:
Because of their transmitted memory and short individual lifespan, Korath do not have the same relationship to time as we do.  I've suggested on a couple occasions that one of the quirks I'd like to see in their speech is always using the present tense.  When it comes to words they're crudely appropriating from other languages, indications of time are not going to be in their vocabulary.

Reply:
I think I've removed all time references from the file. If I missed any, please let me know.